akka-bbb-fsesl/src/main/resources/application.conf

```
@@ -22,7 +22,7 @@ freeswitch {
     esl {
         host="127.0.0.1"
         port=8021
-        password="ClueCon"
+        password="5c2532ed357f5ca0"
```

```suggestion
        password="ClueCon"
```

Let's keep it as the default in case someone match-replaces it with `sed`

---

My last commit was supposed to revert that. I don't know why it didn't.